03 Design/OCs/OCGetHandles.md

```diff
@@ -0,0 +1,19 @@
+#### Operation: 
+
+getHandles() 
+
+#### Cross reference: 
+
+Use case 1: Send Request 
+
+#### Preconditions: 
+
+Wall wall exists.
+
+List wall.listOfHandles exists. 
```

Comment:
Opdel List wall.listOfHandles of list
List listOfHandles exists.
wall.listOfHandles exists.